### DISCUSSION TITLE
CockroachDB?

### DISCUSSION THREAD

**Discussion by DanielJonesEB** [concourse/concourse#6519] [General]
Hey all! Has anyone ever assessed whether Concourse would work when using CockroachDB in Postgres-compatibility mode? I'm guessing probably not, but I figured it was worth asking in case anyone had already done the leg work in working out how much would need to change.

**Comment by aoldershaw** [comment]
Haven't looked too closely, but based on https://www.cockroachlabs.com/docs/stable/postgresql-compatibility.html, the only thing I can think of off-hand are the triggers on pipeline creation/deletion to manage the pipeline_build_events_x partition tables.

Also, can't tell if CockroachDB's Postgres compatibility mode supports LISTEN/NOTIFY - https://github.com/cockroachdb/cockroach/issues/41522 is still open, but it's not mentioned in the above link. But anyway, LISTEN/NOTIFY is used pretty commonly in Concourse

**Reply by DanielJonesEB** [reply]
Cool, thanks for thinking about it. I'll let you know if we get to explore it ourselves.

**Reply by HemangLavana** [reply]
I would also be interested in this capability.  Do let us know how it goes.
